x-pack/plugins/core_telemetry/server/lib/tasks/index.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IHapiServer } from '../../../';
+import { PLUGIN_ID, VIS_TELEMETRY_TASK } from '../../../constants';
+import { visualizationsTaskRunner } from './visualizations/task_runner';
+
+export function registerTasks(server: IHapiServer) {
+  const { taskManager } = server;
+
+  taskManager.registerTaskDefinitions({
+    [VIS_TELEMETRY_TASK]: {
+      title: 'X-Pack telemetry calculator for Visualizations',
+      type: VIS_TELEMETRY_TASK,
+      numWorkers: 10, // use the max so no other tasks run concurrently
```

Comment:
Maybe we should make `10` a const somewhere that is useable across the project, so we don't have magic numbers all over? I know that numWorkers was configurable at one point, too, so this might not be the max if someone overrides the config. Might be nice to have a module (or maybe have it be part of the taskManager) (e.g. `taskManager.MAX_WORKERS` or something.

Reply:
dad1216e59